universal-application-tool-0.0.1/app/views/admin/versions/VersionListView.java

```diff
@@ -53,6 +53,8 @@ public Content render(List<Version> allVersions, Http.Request request) {
     ImmutableList<Version> olderVersions =
         allVersions.stream()
             .filter(version -> version.getLifecycleStage().equals(LifecycleStage.OBSOLETE))
+            // Sort from newest to oldest. IDs are DB-generated and increment monotonically.
+            .sorted((a, b) -> (int) (b.id - a.id))
```

Comment:
optional suggestion for readability and remove the casting.

.sorted((a, b) -> a.id.compareTo(b.id))
.reversed



should do the tricky here.

Reply:
nice, thank you!